SPARQLWrapper/Wrapper.py

```diff
@@ -702,7 +706,7 @@ def _getRequestEncodedParameters(self, query=None):
         query_parameters = self.parameters.copy()
 
         # in case of query = tuple("query"/"update", queryString)
-        if query and (isinstance(query, tuple)) and len(query) == 2:
+        if query and len(query) == 2:
```

Comment:
This looks like it can go into a separate PR.

Reply:
934e334